src/olympia/constants/licenses.py

```diff
@@ -67,9 +67,19 @@ class LICENSE_CC_BY_SA(_LicenseBase):
     builtin = 17
 
 
+class LICENSE_COPYRIGHT_AR(_LicenseBase):
+    id = 8
+    name = _(u'All Rights Reserved')
+    icons = 'copyr'
+    some_rights = False
+    creative_commons = False
+    url = None
+    builtin = 18
+
+
 PERSONA_LICENSES = (LICENSE_COPYRIGHT, LICENSE_CC_BY, LICENSE_CC_BY_NC,
                     LICENSE_CC_BY_NC_ND, LICENSE_CC_BY_NC_SA, LICENSE_CC_BY_ND,
-                    LICENSE_CC_BY_SA)
+                    LICENSE_CC_BY_SA, LICENSE_COPYRIGHT_AR)
```

Comment:
Hmm, forgot about how this was implemented.  We don't want `PERSONA_LICENSES` to change, so create a new constant `ALL_LICENSES` that is `PERSONA_LICENSES` + `LICENSE_COPYRIGHT_AR`.  Then change `LICENSES_BY_BUILTIN` to use this new constant instead.